edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1125,6 +1123,14 @@ export default {
         }
       }
 
+      if (this.value.cloudProvider === 'aws') {
+        const missingProfileName = this.machinePools.some(mp => !mp.config.iamInstanceProfile);
+
+        if (missingProfileName) {
+          this.errors.push(this.t('cluster.validation.iamInstanceProfileName'));
```

Comment:
I think we need to do something like this, otherwise we don't get the correct values fro quotation marks
 
```suggestion
          this.errors.push(this.t('cluster.validation.iamInstanceProfileName', { }, true));
```

![image](https://user-images.githubusercontent.com/835961/167224416-cf6bd674-1793-4cf6-93a2-5fecfaa46c5a.png)